src/dlsnode/storage/protocol/StorageProtocolLegacy.d

```diff
@@ -15,9 +15,12 @@ module dlsnode.storage.protocol.StorageProtocolLegacy;
 import ocean.transition;
 
 import dlsnode.storage.protocol.model.IStorageProtocol;
+import dlsnode.storage.util.Promise;
```

Comment:
Do we still need the legacy storage protocol btw? (Could be removed in a separate PR.)

Reply:
Right, it's no longer needed, I'll do it in the next PR.

Reply:
https://github.com/sociomantic-tsunami/dlsnode/issues/50